metrics_ci_integration/lib/jenkins/model/jenkins_build.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:ci_integration/jenkins/model/jenkins_build_artifact.dart';
+import 'package:ci_integration/jenkins/model/jenkins_building_job.dart';
+import 'package:equatable/equatable.dart';
+
+/// A class representing a single Jenkins build.
+class JenkinsBuild extends Equatable {
```

Comment:
Maybe we can move the 'model' package into the 'client' package? Will be happy to discuss it with you.